windows_containers/index.adoc

```diff
@@ -0,0 +1,27 @@
+[id="windows-container-overview"]
+= Windows Container Support for OpenShift overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. For more information, see the xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[release notes].
```

Comment:
Is the product _Windows Container Support for OpenShift_ or _Windows Container Support for Red Hat OpenShift_? A quick search using Atom shows only one _Windows Container Support for OpenShift_ and 4 for _Red Hat OpenShift_.

Reply:
The product name for WinC is "Windows Container Support for OpenShift."